### PR TITLE
(update) Owner Side now has personalized data

### PR DESCRIPTION
**Pull request by lschwall**

### CODE REVIEW

File: server/routes/bar.js [hunk 1]
```diff
@@ -100,25 +100,72 @@ barRouter.post('/create', (req, res) => {
...
+  } = req.body.bparams;
+  console.log(req.body.bparams, 'BAR PARAMS')
+  Bar.findAll({ where: { id_owner: ownerId } })
     .then((bar) => {
-      res.status(201).send(bar);
+      if (bar.length > 0) {
+        console.info('BAR ALREADY EXISTS')
+        res.status(500).send('BAR ALREADY EXISTS')
+      } else {
+        console.log('BAR CREATED')
```
Comment (jstrzesz) [inline server/routes/bar.js:119]: remove

Comment (lschwall) [reply server/routes/bar.js:119]: will do

File: server/routes/cb.js [hunk 2]
```diff
@@ -184,24 +184,15 @@ cbRouter.delete('/checkout', (req, res) => {
...
-            console.warn('NO CUSTOMERS FOUND')
-          }
-        })
-        .catch(err => {
-          res.sendStatus(500, 'Error in retrieving customers from C_B table', err)
-        })
-    })
-    .catch(err => {
-      res.sendStatus(500, 'Error in finding ownerId in C_B table', err)
+      const arr = []
```
Comment (jstrzesz) [inline server/routes/cb.js:191]: could refactor this to a map or reduce and do away with a forEach

File: src/components/Owner/CustomerEntry.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import { Grid, Typography } from '@material-ui/core';
+import PhoneIcon from '@material-ui/icons/Phone';
+import axios from 'axios';
+
+
+const CustomerEntry = ({ customer }) => {
+
+    const [customerData, setCustomerData] = useState();
+    const getCustomerData = () => {
+        console.log(typeof customer, 'CUSTOMER')
```
Comment (jstrzesz) [inline src/components/Owner/CustomerEntry.jsx:11]: remove

File: src/components/Owner/CustomerEntry.jsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        axios.get(`/db/customer/getFriendById?customerId=${customer}`)
+            .then(({ data }) => {
+                console.log(data, 'DATA')
```
Comment (jstrzesz) [inline src/components/Owner/CustomerEntry.jsx:14]: remove

File: src/components/SignIn/owner/OwnerCredentials.jsx [hunk 5]
```diff
@@ -16,6 +17,7 @@ function OwnerCredentials({ setViewValue }) {
         if (data === 'Email or Password Incorrect') {
           console.info('TRY AGAIN');
         } else {
+          console.log(data)
```
Comment (jstrzesz) [inline src/components/SignIn/owner/OwnerCredentials.jsx:20]: remove

**Review by jstrzesz — CHANGES_REQUESTED** [review]
see comments

**Comment by lschwall** [comment]
broke in code

**Pull request closed without merging**